src/utils/config.js

```diff
@@ -0,0 +1,14 @@
+// unique id of the Google sheet that imports desired columns from the rent-strike form responses public sheet
```

Comment:
We need to check that all of this is current on the design doc.

Reply:
Agreed, good to have this documented.